src/App.css

```diff
@@ -1,3 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 .App {
-  text-align: center;
+  padding: 6rem 2rem;
```

Comment:
- [ ] Please kindly remove padding from `.App` and add `min-height: 100vh;` to make it full page app 👋🏼